ConnectionError occurred at 19/02/2024, 15:36:03 - HTTPSConnectionPool(host='api.coinmarketcap.com', port=443): Max retries exceeded with url: /data-api/v3/cryptocurrency/listing?start=1&limit=100&sortBy=market_cap&sortType=desc&cryptoType=all&tagType=all&audited=false (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x000001DCC805B9E0>: Failed to resolve 'api.coinmarketcap.com' ([Errno 11001] getaddrinfo failed)"))
ConnectionError occurred at 21/02/2024, 12:08:20 - HTTPSConnectionPool(host='api.coinmarketcap.com', port=443): Max retries exceeded with url: /data-api/v3/cryptocurrency/listing?start=1&limit=100&sortBy=market_cap&sortType=desc&cryptoType=all&tagType=all&audited=false (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x000001ABA57E5850>: Failed to resolve 'api.coinmarketcap.com' ([Errno 11001] getaddrinfo failed)"))
ConnectionError occurred at 23/02/2024, 14:20:47 - HTTPSConnectionPool(host='api.coinmarketcap.com', port=443): Max retries exceeded with url: /data-api/v3/cryptocurrency/listing?start=1&limit=100&sortBy=market_cap&sortType=desc&cryptoType=all&tagType=all&audited=false (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x000001EDD09CEAB0>: Failed to resolve 'api.coinmarketcap.com' ([Errno 11001] getaddrinfo failed)"))
ConnectionError occurred at 23/02/2024, 14:21:47 - HTTPSConnectionPool(host='api.coinmarketcap.com', port=443): Max retries exceeded with url: /data-api/v3/cryptocurrency/listing?start=1&limit=100&sortBy=market_cap&sortType=desc&cryptoType=all&tagType=all&audited=false (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x000001EDD09CFF50>: Failed to resolve 'api.coinmarketcap.com' ([Errno 11001] getaddrinfo failed)"))
ConnectionError occurred at 23/02/2024, 21:00:40 - HTTPSConnectionPool(host='api.coinmarketcap.com', port=443): Read timed out.
ConnectionError occurred at 23/02/2024, 21:42:47 - HTTPSConnectionPool(host='api.coinmarketcap.com', port=443): Read timed out.
ConnectionError occurred at 03/03/2024, 11:29:08 - ('Connection aborted.', ConnectionResetError(10054, 'An existing connection was forcibly closed by the remote host', None, 10054, None))
ConnectionError occurred at 03/03/2024, 11:40:13 - HTTPSConnectionPool(host='api.coinmarketcap.com', port=443): Max retries exceeded with url: /data-api/v3/cryptocurrency/listing?start=1&limit=100&sortBy=market_cap&sortType=desc&cryptoType=all&tagType=all&audited=false (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x000001F0FF7C8A40>: Failed to resolve 'api.coinmarketcap.com' ([Errno 11001] getaddrinfo failed)"))
ConnectionError occurred at 03/03/2024, 11:41:24 - HTTPSConnectionPool(host='api.coinmarketcap.com', port=443): Max retries exceeded with url: /data-api/v3/cryptocurrency/listing?start=1&limit=100&sortBy=market_cap&sortType=desc&cryptoType=all&tagType=all&audited=false (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x000001F0FF7C8BC0>: Failed to resolve 'api.coinmarketcap.com' ([Errno 11001] getaddrinfo failed)"))
ConnectionError occurred at 03/03/2024, 11:54:32 - HTTPSConnectionPool(host='api.coinmarketcap.com', port=443): Max retries exceeded with url: /data-api/v3/cryptocurrency/listing?start=1&limit=100&sortBy=market_cap&sortType=desc&cryptoType=all&tagType=all&audited=false (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x000001F0FF7C8E90>: Failed to resolve 'api.coinmarketcap.com' ([Errno 11001] getaddrinfo failed)"))
ConnectionError occurred at 03/03/2024, 12:00:47 - HTTPSConnectionPool(host='api.coinmarketcap.com', port=443): Max retries exceeded with url: /data-api/v3/cryptocurrency/listing?start=1&limit=100&sortBy=market_cap&sortType=desc&cryptoType=all&tagType=all&audited=false (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x000001F0FF7C8DA0>: Failed to resolve 'api.coinmarketcap.com' ([Errno 11001] getaddrinfo failed)"))
TimeoutError occurred at 03/03/2024, 12:01:58 - HTTPSConnectionPool(host='api.coinmarketcap.com', port=443): Read timed out. (read timeout=10)
